apps/docs/src/pages/500.mdx

```diff
@@ -0,0 +1,4 @@
+An unknown error while trying to give you what you asked for. If you think it is
```

Comment:
Same as page 404 : Maybe we can just a title, to see directly that is an error page